src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
+    <ul className="CommentList">
+      {comments.length > 0 ? (
+        comments.map(comment => (
+          <CommentInfo comment={comment} key={comment.id} />))
+      ) : (
+        <p data-cy="NoCommentsMessage">No comments yet</p>)}
```

Comment:
Add a space before the closing curly brace to make the code more readable and consistent 👓